jetstream/defaults/firefox_desktop.toml

```diff
@@ -12,6 +12,8 @@ weekly = [
   "search_count",
   "qualified_cumulative_days_of_use",
   "is_default_browser",
+  "sponsored_pocket_ctr",
```

Comment:
So the intention is to calculate these metrics for every Firefox desktop experiment?

Reply:
It only really applies to newtab experiments so I guess not, changing now